internal/utils/plangenerator.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the Apache License, Version 2.0, see LICENCE file for details.
+
+package utils
```

Comment:
I really dislike the utils package. Can we add this file to a new package called `testing`? Then we can work to get rid of utils.

Reply:
Sure, no problem